deploy.sh

```diff
@@ -27,6 +27,11 @@ export BITOPS_CONFIG_SCHEMA="$CLOUDFORMATION_ROOT_SCRIPTS/bitops.schema.yaml"
 
 export SCRIPTS_DIR="$CLOUDFORMATION_ROOT_SCRIPTS/scripts"
 
+if [ "$SKIP_DEPLOY_CFN" == "true" ] || [ "$CFN_SKIP_DEPLOY" == "true" ]; then
```

Comment:
Why `SKIP_DEPLOY_CFN` is used? Are we referring anywhere to that variable?

I see `SKIP_DEPLOY_CLOUDFORMATION` was used in the past per https://bitops.sh/tool-configuration/configuration-cloudformation/#cfn-files

So the ENV var confusion is present. We need to spend extra cycles trying to keep those in sync.